rest-countries-API/script.js

```diff
@@ -0,0 +1,3 @@
+fetch("https://restcountries.com/v3.1/all")
+  .then((res) => res.json())
+  .then(console.log);
```

Comment:
This code block was added just to check whether the API fetch is working properly or not. Further, I'll be adding code blocks to make it more interactive